<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>joueurs</class>
 <widget class="QDialog" name="joueurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>577</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-color: rgb(255, 255, 255);

selection-background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="impression_badges">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>941</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(202, 202, 202);
</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>S'authentifier</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>150</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-5</y>
       <width>501</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../v.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 235, 239);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../ACCOUNT.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>label_10</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>gestion joueur</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(32, 11, 148);
selection-background-color: rgb(35, 5, 132);
background-color: rgb(17, 5, 147);</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(15, 59, 255);</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_afficher">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>afficher</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>année de naissance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prenom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_annees_naiss">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nationalité</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nationalite">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_importer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>270</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>impoter photo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>type du sport</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_type_sport">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-30</y>
       <width>941</width>
       <height>541</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bb.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableView" name="tab_joueur">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>31</y>
       <width>451</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>exporter PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>calcul score</string>
     </property>
    </widget>
    <widget class="QComboBox" name="trier_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>381</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>trier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prenom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>score</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>année de naissance</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>381</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>rechercher</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nationalité </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>type du sport</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nombre de competition</string>
      </property>
     </item>
    </widget>
    <zorder>label_8</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>le_id</zorder>
    <zorder>pb_ajouter</zorder>
    <zorder>pb_modifier</zorder>
    <zorder>pb_afficher</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>le_nom</zorder>
    <zorder>le_prenom</zorder>
    <zorder>le_annees_naiss</zorder>
    <zorder>label_5</zorder>
    <zorder>le_nationalite</zorder>
    <zorder>pb_importer</zorder>
    <zorder>label_6</zorder>
    <zorder>le_type_sport</zorder>
    <zorder>tab_joueur</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>pushButton_8</zorder>
    <zorder>trier_2</zorder>
    <zorder>comboBox_2</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLineEdit" name="le_id_supp">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>impression et calcul</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>rechercher</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nationalité </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>type du sport</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nombre de competition</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="trier">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>trier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prenom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>score</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>année de naissance</string>
      </property>
     </item>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>291</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>exporter PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>calcul score</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>16</width>
       <height>221</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bb.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_9</zorder>
    <zorder>comboBox</zorder>
    <zorder>trier</zorder>
    <zorder>listView</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>verticalScrollBar</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
